saul-core/src/main/scala/edu/illinois/cs/cogcomp/saul/datamodel/edge/Edge.scala

```diff
@@ -0,0 +1,153 @@
+/** This software is released under the University of Illinois/Research and Academic Use License. See
```

Comment:
I didn't check the content .. hopefully the functionality hasn't change since the diff is not clear.